### PR TITLE
JENKINS-37139 Add support for pipeline jobs

### PR DESCRIPTION
**Pull request by peppelan**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -5,13 +5,18 @@
...
-		<version>1.565.3</version>
+		<version>2.2</version>
 	</parent>
 
 	<artifactId>parameterized-scheduler</artifactId>
 	<version>0.3-SNAPSHOT</version>
 	<packaging>hpi</packaging>
 
+	<properties>
+		<jenkins.version>1.642.3</jenkins.version>
```
Comment (batmat) [inline pom.xml]: It's quite a big jump. I would be ready to bump to the first pipeline compatible version 1.580.1 without much debate, but for such a bump I would prefer some justification.

File: pom.xml [hunk 2]
```diff
@@ -5,13 +5,18 @@
...
+		<maven-hpi-plugin.disabledTestInjection>true</maven-hpi-plugin.disabledTestInjection>
```
Comment (batmat) [inline pom.xml]: Why?

**Comment by leewin12** [comment]
@batmat Jenkins pipeline is getting more common. Could you tell us a plan to next version with this feature?

**Comment by leewin12** [comment]
I fixed parameter missing after refresh page and migrated to 1.642.3 without skipping jenkins tests.
I sent that pull request to you @peppelan,  so please take a look and apply to this PR.
Thank you both of you.

**Comment by peppelan** [comment]
Abandoning pull request in favor of [leewin12's PR](https://github.com/jenkinsci/parameterized-scheduler-plugin/pull/3)

**Pull request closed without merging**